doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -1,383 +1,524 @@
 
-e.g. **point_cloud** parameters:
+By default the MoveIt Setup Assistant assumes **position** command interface
+and **position** and **velocity** state interfaces, and we will proceed with this setting.
 
-.. image:: setup_assistant_panda_3d_perception_point_cloud.png
+.. image:: ros2_control/setup_assistant_ros2_control_tags.png
    :width: 700px
 
-For more details about those parameters please see perception pipeline :doc:`tutorial </doc/examples/perception_pipeline/perception_pipeline_tutorial>`
+If necessary, select the desired command or state interfaces for your robot joints and
+then click the **Add Interfaces** button.
 
-In case of **sensors_3d.yaml** was not needed, choose **None**.
+Step 9: ROS 2 Controllers
+-------------------------
 
-.. image:: setup_assistant_panda_3d_perception.png
+ROS 2 Control is a framework for real-time control of robots,
+designed to manage and simplify the integration of new robot hardware.
+For more details, please look at `ros2_control <https://control.ros.org/master/index.html>`_ documentation.
+
+**ROS 2 Controllers** pane can be used to auto generate simulated controllers to actuate the robot joints.
+
+.. image:: ros2_controllers/setup_assistant_ros2_controllers.png
    :width: 700px
 
+Add the arm controllers
 
-Step 9: Gazebo Simulation
---------------------------
+* Click on the **ROS 2 Controllers** pane selector.
+
+* Click on **Add Controller** and you should see the following screen:
+
+* We will first add Panda arm joint trajectory controller.
+
+* Enter **Controller Name** as ``panda_arm_controller``.
 
-The Simulation tab can be used to help you simulate your robot with Gazebo by generating a new Gazebo compatible urdf if needed.
+* Choose **joint_trajectory_controller/JointTrajectoryController** as the controller type
 
-.. image:: setup_assistant_panda_simulation_screen.png
+.. image:: ros2_controllers/setup_assistant_panda_arm_ros2_controller_type.png
    :width: 700px
 
-You can use the generated robot **urdf** to spawn the robot in Gazebo in the following way.
+* Next, we need to choose the controller joints. Joints can be added individually or by move group.
 
-* Use rosrun to start gazebo empty world: ::
+* Now, click on **Add Planning Group Joints**.
 
-   roslaunch gazebo_ros empty_world.launch paused:=true use_sim_time:=false gui:=true throttled:=false recording:=false debug:=true
+* Choose ``panda_arm`` group from the **Available Groups** tab and add it to the **Selected Groups**.
 
-* Use rosrun to spawn the robot: ::
+* Click **Save** to save the selected controller.
 
-   rosrun gazebo_ros spawn_model -file </path_to_new_urdf/file_name.urdf> -urdf -x 0 -y 0 -z 1 -model panda
+.. image:: ros2_controllers/setup_assistant_panda_arm_ros2_controller_group.png
+   :width: 700px
+
+Add the hand controllers
 
+* Follow the same steps for the arm, but choose **position_controllers/GripperActionController**
```

Comment:
Instead of saying "follow the same steps for the arm", should we just list out the steps? Because notably what is missing from this section is what name to give this new controller (people can read it from the screenshot, I guess, but still...)

Reply:
Yes, this would be improve the clarity. I listed the steps. Thanks for the suggestion!